torch/distributed/_functional_collectives.py

```diff
@@ -1,12 +1,11 @@
 import warnings
-import weakref
 import sys
 import torch
 import torch.distributed as dist
 import torch.distributed.distributed_c10d as c10d
 from typing import Tuple, Union, List, cast, TYPE_CHECKING
 from torch.utils._pytree import tree_map_only
-
+from ._functional_collectives_impl import _all_reduce, _register_wrapper_tensor
```

Comment:
Curious why we only need to import `_all_reduce` impl? I thought we need to import all the collective kernel impls.

Reply:
technically we don't need to import any.

I import it so that i can query it for its '__module__' and pass that module obj to the registration code, which will search/import all the ops by name from that module.

I could have used the `_register_tensor_wrapper` obj to get that module but for some reason i felt its cleaner to use the module that has one of the collectives in it.

Reply:
I see, in this case, can we just import the module instead? i.e. `import ._functional_collectives_impl as funcol_impl`, we can then pass the `funcol_impl` module for registration?